target_definition_files/nirda_readout_schemes.json

```diff
@@ -15,7 +15,7 @@
         "FixedParameters": {
             "AverageGroups": 1,
             "IncludeFieldSolnsInResp": 1,
-            "ROI_StartX": 0,
+            "ROI_StartX": 1968,
```

Comment:
Double checking, (1968, 824) puts the subarray at the edge of the detector, is that correct?

Reply:
Yes - this is 1968 = 2048 - 80.